.github/workflows/push.yml

```diff
@@ -367,6 +360,59 @@ jobs:
       - name: Index Linux packages
         run: ./build/package/linux/index.sh /s3bucket
 
+  update-repositories-windows:
+    needs:
+      - set-version
+      - release
+      - release-msi-windows
+    runs-on: windows-latest
+    if: startsWith(github.ref, 'refs/tags/') && needs.set-version.outputs.is_semantic_tag == 'true'
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Download Scoop manifest
+        uses: actions/download-artifact@v2
+        with:
+          name: scoop-manifest
+          path: target/scoop
+      - name: Push Scoop manifest
+        uses: cpina/github-action-push-to-another-repository@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.SCOOP_RELEASE_GITHUB_PERSONAL_ACCESS_TOKEN }}
+        with:
+          source-directory: "target/scoop"
+          destination-github-username: "keboola"
+          destination-repository-name: "scoop-keboola-cli"
+          target-branch: main
+      - name: Push Chocolatey
+        env:
+          CHOCOLATEY_KEY: ${{ secrets.CHOCOLATEY_KEY }}
+          MSI_CHECKSUM: ${{ needs.release-msi-windows.outputs.msi_checksum }}
+          MSI_FILE: ${{ needs.release-msi-windows.outputs.msi_file }}
+          VERSION: ${{ needs.set-version.outputs.version }}
+        run: |
+          New-Item -ItemType "directory" -Path ".\build\package\chocolatey\push"
+          New-Item -ItemType "directory" -Path ".\build\package\chocolatey\push\tools"
+          cd .\build\package\chocolatey\push
+          $url="$env:DIST_BASE_URL/msi/$env:MSI_FILE"
+          ((Get-Content -path ..\keboola-cli.nuspec -Raw) -replace '{VERSION}',$env:VERSION) | Set-Content -Path .\keboola-cli.nuspec
+          ((Get-Content -path ..\chocolateyinstall.ps1 -Raw) -replace '{URL}',$url -replace '{CHECKSUM}',$env:MSI_CHECKSUM) | Set-Content -Path .\tools\chocolateyinstall.ps1
```

Comment:
Vezmou se templaty těch dvou souborů potřebných pro Chocolatey package (`keboola-cli.nuspec` a `tools\chocolateyinstall.ps1`) a doplní se do nich aktuální číslo verze, url msi souboru a jeho checksum.

Ozkoušel jsem tak že jsem vypnul `choco push` a nahrál ty soubory do artefaktů (https://github.com/keboola/keboola-as-code/actions/runs/1860816795):
Zapomněl jsem nafejkovat env var `VERSION`, tak je tam `dev` ale pro ilustraci že to funguje to stačí. 
![CleanShot 2022-02-17 at 21 32 46@2x](https://user-images.githubusercontent.com/215660/154566028-2732b260-1a5b-482f-8e0a-c950df7aa463.jpg)